Makefile

```diff
@@ -203,3 +204,8 @@ update-gofmt:
 ### update-all:           Update all update- rules.
 .PHONY: update-all
 update-all: update-codegen update-license update-mdlint update-gofmt
+
+### names-check:          Check if e2e test cases' names have the prefix "suite-<suite-name>".
+.PHONY: names-check
```

Comment:
I think `e2e-names-check` is better because it explicitly indicates that it is related to the e2e test.

Reply:
ok, i'll fix it